sitzverteilung-frontend/tests/utility/urlEncoder.spec.ts

```
@@ -13,6 +13,8 @@ import {
 
 import "blob-polyfill";
 
+export const MAX_SHARE_URL_LENGTH = 4096;
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)